admin/admin-model.php

```diff
@@ -140,8 +140,15 @@ public function delete_language( $lang_id ) {
 		// Delete domain
 		unset( $this->options['domains'][ $lang->slug ] );
 
-		// Delete the language itself.
-		foreach ( $lang->get_tax_props( 'term_id' ) as $taxonomy_name => $term_id ) {
+		/**
+		 * Delete the language itself.
+		 *
+		 * Reverses the language taxonomies order is required to make sure 'language' is deleted in last.
+		 *
+		 * The initial order with the 'language' taxonomy at the beginning of 'PLL_Language::term_props' property
+		 * is done by PLL_Model::filter_language_terms_orderby()
+		 */
```

Comment:
`/**` should be used for doc blocks only.
```suggestion
		/*
		 * Delete the language itself.
		 *
		 * Reverses the language taxonomies order is required to make sure 'language' is deleted in last.
		 *
		 * The initial order with the 'language' taxonomy at the beginning of 'PLL_Language::term_props' property
		 * is done by {@see PLL_Model::filter_language_terms_orderby()}.
		 */
```

Reply:
Done in 3dcac07b883dfbb9ab76ddf0e720012a007340b8